apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java

```diff
@@ -55,6 +56,8 @@ public interface DatasetItemDAO {
     Flux<DatasetItem> getItems(UUID datasetId, int limit, UUID lastRetrievedId);
 
     Mono<List<WorkspaceAndResourceId>> getDatasetItemWorkspace(Set<UUID> datasetItemIds);
+
+    Flux<DatasetItemSummary> findDatasetItemSummaryByDatasetIds(Collection<UUID> datasetIds);
```

Comment:
Really minor: datasetIds better as Set. They better be unique and we don't care about the provided order.

Reply:
Updated